projectgenerator/gui/options.py

```diff
@@ -129,4 +129,20 @@ def focusInEvent(self, e):
 
     def mouseReleaseEvent(self, e):
         super(CompletionLineEdit, self).mouseReleaseEvent(e)
-        self.punched.emit()
\ No newline at end of file
+        self.punched.emit()
+
+
+class ModelListView(QListView):
+
+    spaced = pyqtSignal(QModelIndex)
```

Comment:
Interesting word. Does it mean `space_pressed`?

Reply:
Sorry, I guess my liguaphile alter-ego overcame me. I will change it for the good of understandable code..